sesman/chansrv/chansrv_config.c

```diff
@@ -286,10 +320,34 @@ config_dump(struct config_chansrv *config)
     g_writeln("    UseUnixSocket (derived):   %s",
               g_bool2text(config->use_unix_socket));
 
+    char buf[256];
     g_writeln("\nSecurity configuration:");
-    g_writeln("    RestrictOutboundClipboard: %s",
-              g_bool2text(config->restrict_outbound_clipboard));
-
+    if (config->restrict_outbound_clipboard == 0)
+    {
+        g_writeln("    RestrictOutboundClipboard:       %s", "false");
+    }
+    else if (config->restrict_outbound_clipboard == CLIP_RESTRICT_ALL)
+    {
+        g_writeln("    RestrictOutboundClipboard:       %s", "true");
+    }
+    else
+    {
+        g_bitmask_to_str(config->restrict_outbound_clipboard, clip_restrict_map, ',', buf, 256);
```

Comment:
suggest `sizeof(buf)` rather than `256` for maintenance reasons. Same comment for line 348.

Reply:
Fixed.